samples/mini-games/settings.gradle.kts

```diff
@@ -0,0 +1,13 @@
+includeBuild("../../") {
```

Comment:
@chippmann I know you don't like this, but this will help us iterate faster. Let's decide in removing it once we have a stable version.

Reply:
Agree. Let's discuss this again once the merge is done :-) For now though i'm fine with it.